initfiles/componentfiles/configxml/sparkThor.xsd

```diff
@@ -108,52 +88,18 @@
                     </xs:appinfo>
                 </xs:annotation>
             </xs:attribute>
-            <xs:attribute name="HADOOP_CONF_DIR" type="xs:string">
+            <xs:attribute name="NodeGroup" type="xs:string" use="required" default="mythor">
                 <xs:annotation>
                     <xs:appinfo>
-                        <tooltip>Configuration directory for hadoop</tooltip>
+                        <tooltip>Thor group for workers to attach to</tooltip>
                     </xs:appinfo>
                 </xs:annotation>
             </xs:attribute>
-            <xs:attribute name="SPARK_LOCAL_IP" type="xs:nonNegativeInteger">
+            <xs:attribute name="daliServers" type="daliServersType" use="required">
```

Comment:
can this be inherited from the thor cluster config or from the overall environmen?